webplatformcompat/jinja2/webplatformcompat/feature-basic.html

```diff
@@ -119,7 +119,7 @@
           {%- endif %}
         {%- else %}
 
-        ({{ support.support}})
+        (unknown)
```

Comment:
The compatibility data support i10n, but not the API views. Probably deserves a bug for prioritization and discussion.

I'd prioritize it low (and I have, obviously), because so many of these views are prototypes until https://github.com/mdn/browsercompat-contribution is ready, and then will be removed. However, @stephaniehobson and I aren't sure about the right way to do i10n in an Ember.js app (and, sometimes we discuss not doing it in Ember.js). Any advice is welcome.